tasks/main.yml

```diff
@@ -29,5 +29,6 @@
     name: "{{ item.key }}"
     password: "{{ item.value['password'] | default(omit) }}"
     priv: "{{ item.value['privileges'] | default(omit) }}"
+    host: "{{ item.value['host'] | default(omit) }}"
```

Comment:
You say the host needs to be included, is there a reasonable default we can use here? Would `localhost` work?

Reply:
the documentation states that the default of the host key is "localhost", so ansible handles that for us.
https://docs.ansible.com/ansible/latest/modules/mysql_user_module.html#parameter-host

we could change it, but there is no need